inc/saihostif.h

```diff
@@ -230,6 +230,12 @@ typedef enum _sai_hostif_trap_type_t
      */
     SAI_HOSTIF_TRAP_TYPE_PTP_TX_EVENT = 0x00000011,
 
+    /**
+     * @brief DHCP traffic (UDP ports 67, 68)
+     * (default packet action is drop)
```

Comment:
shouldn't the default be forward? The other switch traps are by default dropped since they are link local.

Reply:
default changed to forward